content/en/docs/ops/configuration/multi-revision/gateways/index.md

```diff
@@ -0,0 +1,217 @@
+    # filename: control-plane.yaml
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    metadata:
+      name: control-plane # REQUIRED
+    spec:
+      profile: default
+      components:
+        ingressGateways:
+          - name: istio-ingressgateway
+            # MUST BE DISABLED
+            enabled: false
+    {{< /text >}}
+
+1.  Create a separate `IstioOperator` CR for the gateway(s), ensuring that it has a name and has the `empty` profile:
+
+    {{< text yaml >}}
+    # filename: gateways.yaml
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    metadata:
+      name: gateways # REQUIRED
+    spec:
+      profile: empty # REQUIRED
+      components:
+        ingressGateways:
+          - name: istio-ingressgateway
+            enabled: true
+    {{< /text >}}
+
+1.  Install the `CR`s:
+
+    {{< text bash >}}
+    $ istioctl install -f control-plane.yaml --revision 1-8-0
+    $ istioctl install -f gateways.yaml --revision 1-8-0
+    {{< /text >}}
+
+Istioctl install and the operator track resource ownership through labels for both the revision and owning CR name.
+Only resources whose name and revision labels match the `IstioOperator` CR passed to `istioctl` install/operator will be
+affected by any changes to the CR - all other resources in the cluster will be ignored.
+It is important to make sure that each `IstioOperator` installs components that do not overlap with another `IstioOperator`
+CR, otherwise the two CR's will cause controllers or `istioctl` commands to fight.
+
+### Upgrade with `istioctl`
+
+Let's assume that the target version is 1.8.1.
+
+1.  Download the Istio 1.8.1 release and use the `istioctl` from that release (here called `release-1.8.1/istioctl`)
+to install the Istio 1.8.1 control plane:
+
+{{< text bash >}}
+$ release-1.8.1/istioctl install -f control-plane.yaml --revision 1-8-1
+{{< /text >}}
+
+(Refer to the canary upgrade docs for more details on steps 2-4.)
+
+1.  Verify that the control plane is functional.
+1.  Label workload namespaces with istio.io/rev=1-8-1 and restart the workloads.
+1.  Verify that the workloads are injected with the new proxy version and the cluster is functional.
+1.  At this point, the ingress gateway is still 1.8.0. You should see the following pods running:
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system --show-labels
+
+NAME                                    READY   STATUS    RESTARTS   AGE   LABELS
+istio-ingressgateway-65f8bdd46c-d49wf   1/1     Running   0          21m   service.istio.io/canonical-revision=1-8-0 ...
+istiod-1-8-0-67f9b9b56-r22t5            1/1     Running   0          22m   istio.io/rev=1-8-0 ...
+istiod-1-8-1-75dfd7d494-xhmbb           1/1     Running   0          21s   istio.io/rev=1-8-1 ...
+{{< /text >}}
+
+As a last step, upgrade any gateways in the cluster to the new version:
+
+{{< text bash >}}
+$ release-1.8.1/istioctl install -f gateways.yaml --revision 1-8-1
+{{< /text >}}
```

Comment:
You need to indent all this by 4 spaces, or the list ordering for the next step will go back to 1.
```suggestion
    {{< text bash >}}
    $ kubectl get pods -n istio-system --show-labels

    NAME                                    READY   STATUS    RESTARTS   AGE   LABELS
    istio-ingressgateway-65f8bdd46c-d49wf   1/1     Running   0          21m   service.istio.io/canonical-revision=1-8-0 ...
    istiod-1-8-0-67f9b9b56-r22t5            1/1     Running   0          22m   istio.io/rev=1-8-0 ...
    istiod-1-8-1-75dfd7d494-xhmbb           1/1     Running   0          21s   istio.io/rev=1-8-1 ...
    {{< /text >}}

    As a last step, upgrade any gateways in the cluster to the new version:

    {{< text bash >}}
    $ release-1.8.1/istioctl install -f gateways.yaml --revision 1-8-1
    {{< /text >}}
```